scss/_utilities_has-icon.scss

```diff
@@ -0,0 +1,7 @@
+@mixin vf-u-has-icon {
+  // stylelint-disable-next-line selector-max-type
+  .u-has-icon i:first-child {
```

Comment:
Maybe use the direct child combinator selector to make sure we avoid any unforeseen side effects for next children
```css
    .u-has-icon i:first-child {
```

Reply:
Sorry, I had that in place at the beginning, but the linter was unhappy so I changed stuff trying to make it happy. Then it ended with the disable-next-line, but I forgot to put back what I had before... Anyway, I just pushed the change!